apps/site/lib/site_web/views/partial/svg_icon_with_circle.ex

```diff
@@ -97,7 +97,6 @@ defmodule SiteWeb.PartialView.SvgIconWithCircle do
   defp icon_name(:access), do: "accessible"
   defp icon_name(:no_access), do: "not-accessible"
   defp icon_name(:the_ride), do: "the-ride"
-  defp icon_name(:parking_lot), do: "parking"
```

Comment:
Does content know you're removing this icon?

Reply:
I think so. The CMS uses this reference: https://github.com/mbta/dotcom/blob/39a294dc4b05e68ea0e32416206640c4f4db62a6/apps/site/lib/site/content_rewriters/liquid_objects.ex#L128

So if that's already switched over we should be good.

Reply:
Nothing is being removed. This is a private function only being called from one place which has a guard on the input. It's not possible for this definition to ever be reached.